### ISSUE TITLE
Incompatibility between PyPI and Conda

### ISSUE THREAD

**Issue by jordan-heemskerk** [conda-forge/quaternion-feedstock#79]
### Comment:

It's slightly annoying that this package is called `numpy-quaternion` on PyPI and `quaternion` on conda.

However, not sure on the best solution. Could we stand up a duplicate package in conda called `numpy-quaternion`?

**Comment by moble** [comment]
Yeah, “numpy-quaternion” is unfortunate, but unavoidable due to history.  But I don’t see how making that the conda name would be helpful.  Please explain if I’m missing something.

Right now, conda users probably won’t encounter any confusion, because “quaternion” is the name they install and the name they import.  They shouldn’t have to install with pip, so they don’t even need to know the name.  Pip users are the only ones who will encounter confusion.

**Comment by jordan-heemskerk** [comment]
My use case (yes, a little contrived for sure) is that I want to be able to specify and share a set of requirements for a Python project in a requirements.txt, but my organization is a mix of conda users and pip users, which usually isn't an issue, but for this package it is.

**Comment by moble** [comment]
It's not very contrived.  In fact, I've run into similar situations myself.  I was always able to just have a script that renamed the package if conda was going to use it.  But I can see how that wouldn't work for everyone.  Sorry.  I've long wanted that PyPI name.  :/

I'll also just point out [the `quaternionic` package](https://github.com/moble/quaternionic), which is my newer version that may be easier to use, and is named the same thing on both PyPI and conda-forge.

As for the possibility of making some sort of `numpy-quaternion` alias in conda-forge, I'm afraid that's beyond my knowledge level.  You might want to ping one of the admins.

**Comment by jordan-heemskerk** [comment]
> I've long wanted that PyPI name. :/

LOL. Yeah that seems unfortunate; I'm guessing they will never remove or let you take over the name for backwards compat reasons...

Long-term, moving the quaternionic package seems like a great solution for me. However, in the meantime it would be awesome to get an alias or duplicate set up on conda-forge called `numpy-quaternion`, if you are onboard for that. 

@conda-forge/help-python can anyone advise on if there is a desired way to achieve that effect (or if it's even allowed?)
